modules/3d-tiles/test/lib/parsers/batched-model-3d-tile.spec.js

```diff
@@ -83,6 +83,13 @@ test('batched model tile#with batch table', async t => {
   t.end();
 });
 
+test('batched model tile#gltfUpAxis is supported', async t => {
+  const tileData = await loadRootTileFromTileset(t, WITH_BATCH_TABLE_URL);
+  const tile = await parse(tileData, Tiles3DLoader);
+  t.equal(tile.gltfUpAxis, 'Y', 'tile has default gltf up axis');
+  t.end();
```

Comment:
Ideally I'd like to see that the matrix is affected. Not sure how easy this is to test, it may require more work, like instantiating a Tileset3D class.

Reply:
Added few test cases that check rotation matrix for Y and Z cases. 
In case of Z used "dummy" tileset.